core/pom.xml

```diff
@@ -8,7 +8,6 @@
 	</parent>
 
 	<artifactId>behemoth-core</artifactId>
-	<version>1.2-SNAPSHOT</version>
```

Comment:
This is unrelated to the Mahout upgrade and should be done separately. 
I'd rather keep the versions decoupled even if they are the same, a component could evolve independently of the core or other modules